install.sh

```diff
@@ -55,15 +55,17 @@ apt-get install -y -q \
 
 pip install virtualenv
 
-echo "================= Installing Git ==================="
+export GIT_VERSION=1:2.*
+echo "================= Installing Git $GIT_VERSION ==================="
 add-apt-repository ppa:git-core/ppa -y
 apt-get update -qq
-apt-get install -y -q git=1:2.*
+apt-get install -y -q git="$GIT_VERSION"
 
-echo "================= Adding JQ 1.5.1 ==================="
-apt-get install -y -q jq=1.5*
+export JQ_VERSION=1.5*
+echo "================= Adding JQ $JQ_VERSION ==================="
+apt-get install -y -q jq="$JQ_VERSION"
 
-echo "================= Installing Node 8.x ==================="
+echo "================= Installing Node $NODE_VERSION ==================="
```

Comment:
Isn't the same header being printed from inside the actual install script for node? We should not print it here. Also, the env isn't even defined here. Same thing for all the other stuff that's installed by just calling install.sh.

Reply:
ok